CHANGE_LOG

```diff
@@ -1,3 +1,77 @@
+Version 0.45.0
+--------------
+
+In this release, Numba gained an experimental `numba.typed.List <pending_link>`_ container as a future replacement of the `reflected list <pending_link>`_. In addition, functions decorated with ``parallel=True`` can now be cached to reduce compilation overhead due to the auto-parallelization the function.
+
+
+Enhancements from user contributed PRs (with thanks!):
+
+James Bourbeau added Numba version in the error message in #4227, added the ``signature`` parameter to ``inspect_types`` in #4200, and
+improved the docstring of ``normalize_signature`` in #4205.
+
+Guilherme Leobas implemented the dominator tree and dominance frontier algorithms in #4216 and #4149
+
+Nick White fixed the issue with ``round`` in the CUDA target in #4137.
+
+Joshua Adelman added support for ``x in range(...)`` in #4129.
+
+
+General Enhancements:
```

Comment:
This seems to be missing a bunch of PRs...
```
$ git log 0.45.0dev0..HEAD|grep -i "Merge pull"|egrep -i "#[0-9][0-9]+"|wc -l
```
shows 60, I manually cross referenced these against those on the milestone (which was not complete in itself, but is now).

Reply:
my script stopped prematurely.  the list of PR has been fixed now

Reply:
Can't see these anywhere:
- #4145
- #4151
- #4155
- #4211
?

Also, I just noticed that this proposed change log update is stylistically inconsistent with the records since 0.35.0, from then on the PRs are ordered from lowest ticket number to highest, this one does highest to lowest.